pkg/oauth/registry/helpers/strategy.go

```diff
@@ -0,0 +1,19 @@
+package helpers
+
+import (
+	kapi "k8s.io/kubernetes/pkg/api"
+	"k8s.io/kubernetes/pkg/runtime"
+	"k8s.io/kubernetes/pkg/util/validation/field"
+)
+
+var CannotCreateStrategy = readOnly{kapi.Scheme}
```

Comment:
I prefer having explicit strategy per resource. That's what we do with all of them.